support/k8s/k8s-workload-registrar/config.go

```diff
@@ -85,6 +87,19 @@ func (c *CommonMode) ParseConfig(hclConfig string) error {
 	if c.DisabledNamespaces == nil {
 		c.DisabledNamespaces = defaultDisabledNamespaces()
 	}
+	// eliminate orphaned context
```

Comment:
Need to update the logic on lines 81-83 of the new version so that only of PodLabel, PodAnnotation, or IdentityTemplate is set.

Reply:
agree

Reply:
I moved this check into config_crd.go since it's CRD relevant only